src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -571,12 +572,17 @@
 			.description("Called when a horse jumps.")
 			.examples("on horse jump:", "\tpush event-entity upwards at speed 2")
 			.since("INSERT VERSION");
-		if(Skript.classExists("org.bukkit.event.block.BlockFertilizeEvent"))
+		if (Skript.classExists("org.bukkit.event.block.BlockFertilizeEvent"))
```

Comment:
While I understand this space needs to be here, this should not be part of this PR. Please revery this change.
Changes should only be done which are in regards to the issue/change at hand.

Reply:
Oops, you're right. Will be reverted.